docs/sdk_benchmarks.md

```diff
@@ -0,0 +1,35 @@
+# SDK Usage Benchmarks
+
+## Concept
+
+The way we procured the data for our benchmarks was by profiling several popular open source Android applications 
+([GoogleIO](https://github.com/google/iosched), [RedReader](https://github.com/QuantumBadger/RedReader), [Fenix](https://github.com/mozilla-mobile/fenix)). 
+Same use case was performed on each application with and without Datadog SDK enabled.
```

Comment:
How long was the scenario you used ? Maybe you should add something along the lines of 
> We performed the same use case scenarios lasting 15 minutes, covering the main features of the applications, with and without the Datadog SDK enabled.